### DISCUSSION TITLE
Unterschiedlicher Hintergrund?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2002] [Q&A]
![image](https://user-images.githubusercontent.com/84818934/197764985-266dafd9-c6a7-4a71-800b-a987319edf02.png)
![image](https://user-images.githubusercontent.com/84818934/197765043-04bf154a-b2e8-4bd4-ba66-c33a3a01c7fe.png)
![image](https://user-images.githubusercontent.com/84818934/197765069-0e31e405-5652-41db-9eae-1a70e6610694.png)

Wo kann es herkommen?

![jarvis-Fehler Hintergrund bei einem Tab anders](https://user-images.githubusercontent.com/84818934/197765332-579ce735-9203-4260-a2b7-c1ad6bb8d9b0.gif)

**Comment by Zefau** [comment]
Keine Ahnung, würde sagen, dass das was gecached ist. Was sagen andere Browser dazu?

**Comment by mcuiobroker** [comment]
Bei allen anderen Browsern dasgleiche Ergebnis.

**Comment by Zefau** [comment]
Schau mal im DOM. Wie ist die URL? Wo hängt es genau? Auf `body`? Das Hintergrundbild ist keines, was ich mit jarvis ausliefere. Ggf. altes CSS, was jarvis nicht bereinigt? Schau mal im DOM, ob du Hinweise findest.

**Comment by mcuiobroker** [comment]
Gefunden. 
Kommt von einem StateHTML der innen eine eigene css hat.
```
htmlDP = '<!DOCTYPE HTML><html>';
htmlDP = htmlDP + '<head>';
htmlDP = htmlDP + '<title>HUE-Tabelle</title>';
htmlDP = htmlDP + '<link rel ="stylesheet" type="text/css" href="http://192.168.178.89/download/jarvis.css">';
```
